tests/test_http.py

```diff
@@ -1196,7 +1196,7 @@ def test_add_fail_for_over_limit(self):
     from googleapiclient.http import MAX_BATCH_LIMIT
 
     batch = BatchHttpRequest()
-    for i in xrange(0, MAX_BATCH_LIMIT):
+    for i in range(0, MAX_BATCH_LIMIT):
```

Comment:
Might be cleaner to use `six.moves.range` here, e.g. via the following at the top of the module:

```python
from six.moves import range
```